### PR TITLE
feat(nix): avoid volumes for better macos support

### PR DESCRIPTION
**Pull request by hussein-aitlahcen**

### CODE REVIEW

File: .nix/services/hasura.nix [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
+    files = pkgs.linkFarm "context" [
+      {
+        name = "metadata";
+        path = metadata;
+      }
+      {
+        name = "Dockerfile";
+        path = pkgs.writeText "Dockerfile" ''
+          FROM hasura/graphql-engine:v2.12.0-beta.1.cli-migrations-v3
+          COPY metadata /hasura-metadata
```
Comment (dzmitry-lahoda) [inline .nix/services/hasura.nix:13]: https://github.com/NixOS/nixpkgs/blob/578f231a53bfbedfad4b4cc17547f67ed99a72ba/pkgs/build-support/docker/examples.nix#L27 ?

Comment (dzmitry-lahoda) [reply .nix/services/hasura.nix:13]: I guess metadata can be made other derivation which is mounted here?

**Review by dzmitry-lahoda — APPROVED** [review]

File: .nix/services/hasura.nix (hunk 1 above)
Comment (hussein-aitlahcen) [reply .nix/services/hasura.nix:13]: it's arion gere

Comment (hussein-aitlahcen) [reply .nix/services/hasura.nix:13]: Here*

Comment (dzmitry-lahoda) [reply .nix/services/hasura.nix:13]: oh, my bad, sorry :)

Comment (dzmitry-lahoda) [reply .nix/services/hasura.nix:13]: i though they integrated, likely in future:)

Comment (hussein-aitlahcen) [reply .nix/services/hasura.nix:13]: Yes hopefully :). In any case even if we copy store paths you need a way to have the path being copied in a target if the tool has no env variable to set an arbitrary one

**Review by cor — APPROVED** [review]
LGTM

**Pull request merged**